invenio/modules/communities/templates/communities/new_base.html

```diff
@@ -214,5 +220,74 @@ <h3 id="deletemodal_label">Delete community?</h3>
 
 init_ckeditor("page", 'advanced');
 init_ckeditor("description", 'simple');
+
+/**
+ * Module validating FileField
+ * @module FileFieldValidator
+ */
+(function () {
+    "use strict";
+    /** @constant {number}
+     *  @default 1.5MB
+     */
+    var MAX_SIZE = 1.5e+6;
+    /** @constant {string[]} */
+    var COMMUNITIES_LOGO_EXTENSIONS = ['.png', '.jpg', '.jpeg']
+    /**
+     * Validate if given file name is supported.
+     * @param {string} filename - Name of the file to be validated.
+     * @param {string[]} exts - Array of supported extensions.
+     * @returns {boolean} isValid - Flag showing if the file has valid extension.
+     */
+    var hasValidExt = function hasValidExt(filename, exts) {
+        var isValid = false;
+        if (filename.split(".").length >= 2) {
+            var ext = "." + filename.split(".").pop();
+            if (exts.indexOf(ext) >= 0) {
+                isValid = true;
+            }
+        }
+        return isValid;
+    }
+    /**
+     * Convert bytes to human-readable
+     * @param {number} bytes - Size in bytes.
+     * @returns {string} - Specially formatted result.
+     */
+     var bytesToSize =  function bytesToSize( bytes ) {
+       if ( bytes == 0 ) return '0 Byte';
+       var k = 1000;
+       var sizes = ['Bytes', 'KB', 'MB', 'GB'];
+       var i = Math.floor( Math.log(bytes) / Math.log(k) );
+       return ( bytes / Math.pow(k, i) ).toPrecision(2) + ' ' + sizes[i];
+    }
+
+    var logo = document.getElementById( 'logo' );
```

Comment:
- in general we are using JQuery `$` selector
- please `jslint` your code for things like `( 'logo' )` vs. `('logo')`

Thanks